src/test/java/harness/utils/ExternalGCSBuckets.java

```diff
@@ -153,6 +155,14 @@ public static Storage getStorageClient(GoogleCredentials credentials) {
 
   /** Utility method to get the gs:// path of a bucket. */
   public static String getGsPath(String bucketName) {
-    return "gs://" + bucketName;
+    return getGsPath(bucketName, Optional.absent());
+  }
+
+  public static String getGsPath(String bucketName, Optional<String> filePath) {
```

Comment:
Are there any places other than the overloaded method directly above that specify `Optional.empty()` when calling this method? If not, I think it would be cleaner to not require the `Optional` wrapper here and instead have this method call the one above e.g.:
```
return getGsPath(bucketName) + "/" + fileName;
```